<?xml version="1.0" encoding="utf-8"?>
<launch>
	<arg name="veh" default="$(env VEHICLE_NAME)"/>
	<arg name="ai_trafo_mode" default="cb" doc="'cb' for colo balance only; 'both' for color balance and linear trafo"/>
	<arg name="ai_interval" default="5" doc="interval with which the linear trafo gets updated. color balance is performed every second."/>
	<arg name="verbose" default="false"/>

	<!-- start basic args -->
	<include file="$(find duckietown_demos)/launch/master.launch">
		<arg name="veh" value="$(arg veh)"/>
		<arg name="visualization" default="true" />
		<arg name="param_file_name" default="default" />
		<arg name="line_detector_param_file_name" default="default" />
		<arg name="ai_trafo_mode" value="$(arg ai_trafo_mode)"/>
		<arg name="ai_interval" value="$(arg ai_interval)"/>
		<arg name="/lane_following/lane_controller" default="false"/>
		<arg name="/lane_following/stop_line_filter" default="false" /> <!-- will not every allow a transition into COORDINATION mode -->
		<arg name="vehicle_avoidance" default="false"/>
		
	</include>
	
	<group>
		<remap from ="goal_tracking_controller/car_cmd" to ="vehicle_avoidance_control_node/car_cmd"/>
		<remap from ="obstacle_and_goal_filter/compressed" to ="image_transformer_node/corrected_image/compressed"/>
 		<include file="$(find fp_gt_and_oa)/launch/fp_local.launch">
        		<arg name="veh" value="$(arg veh)"/>
        	</include>
        </group>
	
</launch>
